### ISSUE TITLE
Enhancement: Throw Exception if unmapped/unmatched properties

### ISSUE THREAD

**Issue by ghost** [mikependon/RepoDB#427]
It would be very nice to have some kind of option to let RepoDb throw an Exception, if there are some Properties on the class that does not have a column in the database.

Can't figure out if that already possible, or else I find it very useful to actually detect if you get out of sync.

Maybe on RepoDb.SqlServerBootstrap.Initialize(true) = Checking model/database alignment.

Best,
Christian

**Comment by mikependon** [comment]
Hmmmm. It seems to be opposite on the purpose of making this library more dynamic. It was like that before until somebody within Orsted (my current organization) requested it to skip the unmatches propeties/fields :)

**Comment by mikependon** [comment]
But TBH, it is good to have it as a settings. Let me think of it as a setting somewhere (either utilize the Converter class).

**Comment by ghost** [comment]
Yes a setting would be nice, as can see a use case for both behaviors.

I tend to use a model first approach so that the model has God status, if the DB does not honor the (full) model I would like to know :-)

Keep up the good work, it's amazing!!!

Best,
CU

**Comment by mikependon** [comment]
Just leave this enhancement request open, we will assess and will update further soon.

**Comment by mikependon** [comment]
@Kodestuen hey, would you be able of contributor for this?

**Comment by ghost** [comment]
@mikependon hey, I don't think I have the overview/knowledge to accomplish this the right way. 
I have solved it by creating some unit tests that compare objects (Insert/Update/Select) .

Also I found out it's quite strict to have a 100% match in model, so attributes/reflection on the model should be in play to make something usefull.

**Comment by mikependon** [comment]
Do you really require to throw an exception if the property is not matching?
